test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities

```

Comment:
@istoimenova I am talking not about excluding tests from CRQ testing, just about separating tests which tests directly CRQ and regression tests.
For example in test run list describe them like:
./test_scripts/API/IsReady/UI_IsReady/RAI/078_ATF_P_hmi_capabilities_UI_IsReady_message_IsInvalidCharacter_Newline_RAI_SUCCESS.lua REGRESSION_Check_that_SDL_gets_steeringWheelLocation_from_UI_GetCapabilities_when_UI_IsReady_message_IsInvalidCharacter_Newline_RAI_SUCCESS.
(We can discuss it separately with team).
Also, I am do not sure that "new line" separation in test set list will be correctly interpreted by CI.